SU-SWS/react_paragraphs
# Added viewfield widget

# READY FOR REVIEW

# Summary
- Added viewfield widget

# Need Review By (Date)
- asap

# Urgency
- medium

# Steps to Test
1. checkout this branch
1. add `viewfield` module and enable
1. add a paragraph type or edit an existing one and add a viewfield field to the paragraph
1. in a react widget page verify the viewfield works as expected.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
